.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ executors:
   linux-aarch64:
     machine:
       image: ubuntu-2204:current
-    resource_class: arm.medium
+    resource_class: arm.large
```

Comment:
I needed to switch to a `large` VM because the build of Node.js takes more than one hour on `medium` and the job is killed.
If Bioconda team thinks that it should stay `medium` then I'd suggest to merge this PR with `large` so that the recipe is published/released and then revert to `medium` in a follow-up PR. And also add a build instructions to `k8` recipe that it needs `large` for a release.

Reply:
In the meantime the Azure builds started failing:
```
12:47:04 [32mBIOCONDA INFO[0m (OUT)   $BUILD_PREFIX/bin/x86_64-conda-linux-gnu-c++ -o $SRC_DIR/node-v18.19.1/out/Release/obj.target/v8_initializers/gen/torque-generated/third_party/v8/builtins/array-sort-tq-csa.o $SRC_DIR/node-v18.19.1/out/Release/obj/gen/torque-generated/third_party/v8/builtins/array-sort-tq-csa.cc '-D_GLIBCXX_USE_CXX11_ABI=1' '-DNODE_OPENSSL_CONF_NAME=nodejs_conf' '-DNODE_OPENSSL_HAS_QUIC' '-DICU_NO_USER_DATA_OVERRIDE' '-DV8_GYP_BUILD' '-DV8_TYPED_ARRAY_MAX_SIZE_IN_HEAP=64' '-D__STDC_FORMAT_MACROS' '-DOPENSSL_NO_PINSHARED' '-DOPENSSL_THREADS' '-DV8_TARGET_ARCH_X64' '-DV8_HAVE_TARGET_OS' '-DV8_TARGET_OS_LINUX' '-DV8_EMBEDDER_STRING="-node.28"' '-DENABLE_DISASSEMBLER' '-DV8_PROMISE_INTERNAL_FIELD_COUNT=1' '-DV8_ENABLE_PRIVATE_MAPPING_FORK_OPTIMIZATION' '-DV8_SHORT_BUILTIN_CALLS' '-DOBJECT_PRINT' '-DV8_INTL_SUPPORT' '-DV8_ATOMIC_OBJECT_FIELD_WRITES' '-DV8_ENABLE_LAZY_SOURCE_POSITIONS' '-DV8_USE_SIPHASH' '-DV8_SHARED_RO_HEAP' '-DV8_WIN64_UNWINDING_INFO' '-DV8_ENABLE_REGEXP_INTERPRETER_THREADE...
##[warning]Free memory is lower than 5%; Currently used: 95.51%
##[warning]Free memory is lower than 5%; Currently used: 95.51%
```
They were passing last week but apparently the build of Node.js from sources is too heavy for the CI agents...